scripts/deploy/deploy-webapi.ps1

```diff
@@ -42,22 +42,55 @@ if ($LASTEXITCODE -ne 0) {
 }
 
 Write-Host "Getting Azure WebApp resource name..."
-$webappName=$(az deployment group show --name $DeploymentName --resource-group $ResourceGroupName --output json | ConvertFrom-Json).properties.outputs.webapiName.value
-if ($null -eq $webAppName) {
+$deployment=$(az deployment group show --name $DeploymentName --resource-group $ResourceGroupName --output json | ConvertFrom-Json)
+$webApiUrl = $deployment.properties.outputs.webapiUrl.value
+$webApiName = $deployment.properties.outputs.webapiName.value
+
+if ($null -eq $webApiName) {
     Write-Error "Could not get Azure WebApp resource name from deployment output."
     exit 1
 }
 
-Write-Host "Azure WebApp name: $webappName"
+Write-Host "Azure WebApp name: $webApiName"
 
 Write-Host "Configuring Azure WebApp to run from package..."
-az webapp config appsettings set --resource-group $ResourceGroupName --name $webappName --settings WEBSITE_RUN_FROM_PACKAGE="1" | out-null
+az webapp config appsettings set --resource-group $ResourceGroupName --name $webApiName --settings WEBSITE_RUN_FROM_PACKAGE="1" | out-null
 if ($LASTEXITCODE -ne 0) {
     exit $LASTEXITCODE
 }
 
-Write-Host "Deploying '$PackageFilePath' to Azure WebApp '$webappName'..."
-az webapp deployment source config-zip --resource-group $ResourceGroupName --name $webappName --src $PackageFilePath
+Write-Host "Deploying '$PackageFilePath' to Azure WebApp '$webApiName'..."
+az webapp deployment source config-zip --resource-group $ResourceGroupName --name $webApiName --src $PackageFilePath
 if ($LASTEXITCODE -ne 0) {
     exit $LASTEXITCODE
-}
\ No newline at end of file
+}
+
+
+$origin = "https://$webApiUrl"
+Write-Host "Ensuring '$origin' is included in AAD app registration's redirect URIs..."
+
+$webapiSettings = $(az webapp config appsettings list --name $webapiName --resource-group $ResourceGroupName | ConvertFrom-JSON)
```

Comment:
Can you please confirm that this command only returns the name in the output. The webApi settings contains secrets and the output can end up in the github logs.

Reply:
The command returns EVERYTHING in webapiSettings but doesn't print anything on the console.

Is that a problem?

Reply:
Is this two changes in one PR:

1. The deployment update
2. The middleware/options update

Or are they connected?

Reply:
The change consists of distributing CC as one package instead of two (one for the backend and another for the frontend).

This means clicking on "Deploy to Azure" is all you'll need to do to have a complete working deployment (no more additional shell scripts needed).